projects/ngxpert/hot-toast/src/lib/components/hot-toast-group-item/hot-toast-group-item.component.ts

```diff
@@ -122,20 +123,20 @@ export class HotToastGroupItemComponent {
       this.top ? 'Negative' : 'Positive'
     } ${EXIT_ANIMATION_DURATION}ms forwards cubic-bezier(0.06, 0.71, 0.55, 1) var(--hot-toast-exit-animation-delay) var(--hot-toast-exit-animation-state)`;
 
-    const animation = this.toast.autoClose ? `${enterAnimation}, ${exitAnimation}` : enterAnimation;
+    const animation = this.toast().autoClose ? `${enterAnimation}, ${exitAnimation}` : enterAnimation;
 
-    return { ...this.toast.style, animation };
+    return { ...this.toast().style, animation };
   }
 
   get isIconString() {
-    return typeof this.toast.icon === 'string';
+    return typeof this.toast().icon === 'string';
   }
 
   get groupChildrenToastRefs() {
```

Comment:
Instead of getter, create a computed property.

Reply:
done